src/adk/api/local_api.py

```diff
@@ -277,15 +278,64 @@ def get_application_data(application_path: Path) -> ApplicationDataType:
         application_data = utils.read_json_file(manifest_json_file)
         return cast(ApplicationDataType, application_data)
 
+    def init_application(self, application_name: str, application_path: Path) -> None:
+        """
+        Initializes the application directory structure for an existing set of files. The files are moved to the right
+        directory when needed. The files are not created if they do not exist (we have application create command for
+        that). Only manifest.json is created when it doesn't exist or adjusted when it exists. The application name
+        is written/updated to it. The application is added to the application configuration.
+
+        Args:
+            application_name: name of the application
+            application_path: the path where the application is stored
+
+        Raises:
+            DirectoryAlreadyExists: Raised when directory (or file) application_name already exists
+        """
+        application_name = application_name.lower()
+
+        # init the config files
+        app_config_path = application_path / 'config'
+        if not app_config_path.exists():
+            # create config directory and move all config-files to config
+            app_config_path.mkdir(parents=True, exist_ok=True)
+
+        # Move the config files to the config directory
+        utils.move_files(application_path, app_config_path,
+                         files_list=self.__get_config_file_names())
+
+        app_src_path = application_path / 'src'
+        if not app_src_path.exists():
+            # create src and move all python files to src
+            app_src_path.mkdir(parents=True, exist_ok=True)
+
+        # Move all python files to the src directory
+        python_files = self.get_application_file_names(application_path)
+        utils.move_files(application_path, app_src_path,
+                         files_list=python_files)
+
+        # prepare manifest
+        if not (application_path / 'manifest.json').is_file():
+            application_data = utils.get_default_manifest(application_name)
+        else:
+            application_data = self.get_application_data(application_path)
+            application_data["application"]["name"] = application_name
```

Comment:
If there is existing remote information (copied from the earlier app), then that remote information should be reset. 
        # reset remote
        application_data["remote"] = {}

Reply:
Right! Will add the reset.